configure.ac

```diff
@@ -163,7 +163,7 @@ case "${host_cpu}" in
 
     ;;
 
-  aarch64)
+  aarch64|arm64)
```

Comment:
Which platforms use `arm64` instead of `aarch64`?

Reply:
FreeBSD does for example.  I have not checked other platforms.